R/extremes-data.R

```diff
@@ -210,7 +212,27 @@ createDataRows <-
         {
           relatedValue <- relatedSet$value
         }
+        
+        if (!isEmptyOrBlank(x$relatedPrimary)) {
+          if (nrow(x$relatedPrimary) > nrow(x$points) ||
+              nrow(x$points) > nrow(x$relatedPrimary)) {
```

Comment:
Ha, I just mean replace each of those statements, it won't just be one statement.